src/test/regress/expected/multi_materialized_view.out

```diff
@@ -0,0 +1,125 @@
+---
+--- MULTI_MATERIALIZED_VIEW
+---
+-- This file contains test cases for materialized view support. It verifies
+-- various Citus features: 
+-- materialized views work
+-- insert into... select works with views
+CREATE VIEW air_shipped_lineitems AS SELECT * FROM lineitem_hash_part WHERE l_shipmode = 'AIR';
+CREATE TABLE temp_lineitem(LIKE lineitem_hash_part);
+SELECT create_distributed_table('temp_lineitem', 'l_orderkey', 'hash', 'lineitem_hash_part');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+INSERT INTO temp_lineitem SELECT * FROM air_shipped_lineitems;
+SELECT count(*) FROM temp_lineitem;
+ count 
+-------
+  1706
+(1 row)
+
+-- following is a where false query, should not be inserting anything
+INSERT INTO temp_lineitem SELECT * FROM air_shipped_lineitems WHERE l_shipmode = 'MAIL';
+SELECT count(*) FROM temp_lineitem;
+ count 
+-------
+  1706
+(1 row)
+
+-- can create and query materialized views
+CREATE MATERIALIZED VIEW mode_counts
```

Comment:
Can you make sure to add tests with all the  options described in the  postgres docs: https://www.postgresql.org/docs/11/sql-refreshmaterializedview.html

For example, add a test with `refresh materialized view CONCURRENTLY ..`. You may need to create unique index